signing_service/signing_service/signing_service.py

```diff
@@ -458,14 +458,14 @@ def _parse_arguments() -> argparse.Namespace:
 
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        '-host',
-        '--concent_cluster_host',
+        '-ch',
```

Comment:
I am for `-c` if it is not already taken

Reply:
The single-dash flags should be single-letter. It does not make sense to make them longer (and does it even work?). That's what the full flags are for.